tests/checkers/unittest_unicode/unittest_bad_chars.py

```diff
@@ -130,6 +130,7 @@ def test_find_bad_chars(
             # string
             module = astroid.MANAGER.ast_from_string(file)
         except AstroidBuildingError:
+            # pylint: disable-next=redefined-variable-type
```

Comment:
This is potentially a regression, but it could also be something we did not pick up previously and are now incorrectly picking up (if you understand what I mean).

The warning makes _some_ sense, but perhaps we should allow this use of ``cast``

Reply:
Yeap, I think astroid got better so a new false positive appeared in pylint (or is it ? It seems to make sense). Happens often when upgrading astroid in fact 😄

Reply:
Yeah, but I don't understand why it happened `2.10>2.11`. Like, we did nothing for this in `2.11`, right? Well anyway, doesn't hurt too much.

_And I don't think I can take waiting for a 2.11.1 hotfix anymore_

Reply:
>And I don't think I can take waiting for a 2.11.1 hotfix anymore

We definitely need to release soon, there are hundred of issues fixed in 2.13, some of them get reopened as duplicate frequently.